coda/coda_mdstore/tests/test_views.py

```diff
@@ -591,22 +591,42 @@ def test_raises_http404_file_handle_is_falsy(self, rf):
         with pytest.raises(http.Http404):
             views.bagURLList(request, bag.name)
 
-    @pytest.mark.xfail(reason='Refactor is required.')
-    def test_output_text(self):
-        assert 0
-
-    @pytest.mark.xfail(reason='Refactor is required.')
-    def test_top_files_block(self):
-        # TODO: rename once implemented.
-        assert 0
-
-    @pytest.mark.xfail(reason='Refactor is required.')
-    def test_path_is_not_unicode_safe(self):
-        assert 0
+    def test_response_content(self, rf):
+        """Test the output contains domain and parsed tag attributes from getFileHandle."""
```

Comment:
I am confused by this docstring. I am not sure from where parsed tag attributes is coming into this. This test shows itself to be about testing the response from the `bagURLList` view.

Reply:
`bagURLList` uses `getFileList` which parses each `href` tag as a file and adds it to pathList.